docs/mcd.md

```diff
@@ -0,0 +1,329 @@
+
+| Name  | Meaning | Optional? | Value Notes | Other Notes |
+| ----  | ------- | --------- | ----------- | ----------- |
+| enabled | Whether to show the Database selector | No | Allowed values: `true`, `false`| |
+| label | The label shown to the user | Yes | Default value: `Database` | |
+
+### `<field>`
+
+An optional child of `<database>`, this indicates whether the user must provide a value for Database, and a default value for it. It has the following XML attributes. 
+
+| Name  | Meaning | Optional? | Value Notes | Other Notes |
+| ----  | ------- | --------- | ----------- | ----------- |
+| optional | Whether the user must provide a value for Database | Yes | Allowed values: `true`, `false`<br>Default value: `false`| |
+| default-value | A default value for Database | Yes | Default value: `""` | |
+
+### `<schema>`
+
+An optional child of `<connection-metadata>`, this controls whether the Schema selector is shown in the shema viewer. If it is not present, Schema does not appear. It has the following XML attributes. 
+
+| Name  | Meaning | Optional? | Value Notes | Other Notes |
+| ----  | ------- | --------- | ----------- | ----------- |
+| enabled | Whether to show the Schema selector | No | Allowed values: `true`, `false`| |
+| label | The label shown to the user | Yes | Default value: `Schema` | |
+
+### `<table>`
+
+An optional child of `<connection-metadata>`, this controls whether the Table selector is shown in the shema viewer. If it is not present, Table does not appear. It has the following XML attributes. 
+
+| Name  | Meaning | Optional? | Value Notes | Other Notes |
+| ----  | ------- | --------- | ----------- | ----------- |
+| enabled | Whether to show the Table selector | No | Allowed values: `true`, `false`| |
+| label | The label shown to the user | Yes | Default value: `Table` | |
+
+
+## Example
+
+The Connection Metadata file below matches the default--that is, what you get if you don't reference the file in the manifest at all--except that it provides a default value of 'TestV1' for Database. 
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+
+<connection-metadata>
+  <database enabled='true' label='Database'>
+    <field optional='true' default-value='TestV1' />
+  </database>
+  <schema enabled='true' label="Schema" />
+  <table enabled='true' label="Table" />
+</connection-metadata>
+```
+
+The left side image below shows what the Schema Viewer will look like with this Connection Metadata file. The right side shows what it will look like if you remove the `<schema>` element from the file.
+
+<p float="left">
+  <img src="{{ site.baseurl }}/assets/mcd-schema-viewer-1.png" alt="Schema Viewer with Connection Metadata file" align=top hspace=40/>
+  <img src="{{ site.baseurl }}/assets/mcd-schema-viewer-2.png" alt="Schema Viewer after removing schema element from Connection Metadata file" align=top />
```

Comment:
For some reason cannot see the two images

Reply:
Same as other images.